forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/BaseDeviceBindingTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 - 2024 ForgeRock. All rights reserved.
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
```

Comment:
2023 is the correct year here... this test was moved from `auth` module to `integration-tests`